package.json

```diff
@@ -23,14 +24,14 @@
   },
   "homepage": "https://github.com/blackbaud/skyux-progress-indicator#readme",
   "peerDependencies": {
-    "@angular/common": "^4.3.6",
-    "@angular/core": "^4.3.6",
+    "@angular/common": ">=4.3.6",
+    "@angular/core": ">=4.3.6",
```

Comment:
Why change these to >=? Are you sure you want to pull in major versions of angular automatically?

Reply:
It only communicates that this can work with all versions of Angular greater than `4.3.6`, it won't pull them in automatically. See: https://github.com/blackbaud/skyux-core/blob/master/package.json#L26